### PR TITLE
Add bot icon in "other languages" tab in content forms

### PR DESCRIPTION
**Pull request by JoeyStk**
### Short description
This PR adds the bot icon for languages that are in the "Other languages" dropdown

### Proposed changes

- Add changelog entry
- Add if statement for machine translations
- Add translation for it

### Side effects

- I didn't find any

### Resolved issues

Fixes:  #2117

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

File: integreat_cms/cms/templates/_form_language_tabs.html [hunk 1]
```diff
@@ -96,6 +96,11 @@
                                         <span title="{% translate "Translation missing" %}" class="no-trans">
                                             <i icon-name="x"></i>
                                         </span>
+                                    {% elif other_status == translation_status.MACHINE_TRANSLATED %}
+                                        <span title="{% translate "Current machine translation" %}"
```
Comment (timoludwig) [inline integreat_cms/cms/templates/_form_language_tabs.html:100]:
```suggestion
                                        <span title="{% translate "Machine translated" %}"
```
I think the text should be identical to the one in the normal language tab.
If you want to change it here, please also update the text in line 18:
https://github.com/digitalfabrik/integreat-cms/blob/478afa60b99a2166e01cf814c481c6cf2d86b998/integreat_cms/cms/templates/_form_language_tabs.html#L18

File: integreat_cms/release_notes/current/unreleased/2117.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+en: Add bot icon in language dropdown in content forms
+de: Das Bot Icon wird jetzt auch in der Sprachauswahl in den Inhaltformularen gezeigt
```
Comment (timoludwig) [inline integreat_cms/release_notes/current/unreleased/2117.yml:2]:
```suggestion
de: Das Bot-Icon wird jetzt auch im Sprachen-Dropdown in den Inhaltsformularen angezeigt
```

**Review by timoludwig — CHANGES_REQUESTED** [review]

File: integreat_cms/release_notes/current/unreleased/2117.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+en: Add bot icon in language dropdown in content forms
```
Comment (timoludwig) [inline integreat_cms/release_notes/current/unreleased/2117.yml:1]:
```suggestion
en: Add bot icon to language dropdown in content forms
```

File: integreat_cms/cms/templates/_form_language_tabs.html [hunk 4]
```diff
@@ -96,6 +96,11 @@
                                         <span title="{% translate "Translation missing" %}" class="no-trans">
                                             <i icon-name="x"></i>
                                         </span>
+                                    {% elif other_status == translation_status.MACHINE_TRANSLATED %}
+                                        <span title="{% translate "Machine translated" %}"
+                                              class="no-trans">
```
Comment (timoludwig) [inline integreat_cms/cms/templates/_form_language_tabs.html:101]:
```suggestion
                                        <span title="{% translate "Machine translated" %}">
```
Ah, so sorry, and I completely missed the incorrect class :innocent:

**Review by timoludwig — CHANGES_REQUESTED** [review]

File: integreat_cms/cms/templates/_form_language_tabs.html (hunk 4 above)
Comment (JoeyStk) [reply integreat_cms/cms/templates/_form_language_tabs.html]: Thank you!

**Review by timoludwig — APPROVED** [review]
That's it :+1:

**Pull request merged**